api/src/main/java/org/openmrs/obs/handler/ImageHandler.java

```diff
@@ -82,23 +78,10 @@ public Obs getObs(Obs obs, String view) {
 			
 			ComplexData complexData = new ComplexData(file.getName(), img);
 			
-			// Image MIME type
-			try {
-				FileImageInputStream imgStream = new FileImageInputStream(file);
-				Iterator<ImageReader> imgReader = ImageIO.getImageReaders(imgStream);
-				imgStream.close();
-				if (imgReader.hasNext()) {
-					complexData.setMimeType("image/" + imgReader.next().getFormatName().toLowerCase());
-				} else {
-					log.warn("MIME type of " + file.getAbsolutePath() + " is not known");
-				}
-			}
-			catch (FileNotFoundException e) {
-				log.error("Image " + file.getAbsolutePath() + " was not found", e);
-			}
-			catch (IOException e) {
-				log.error("Trying to determine MIME type of " + file.getAbsolutePath(), e);
-			}
+			// Get the Mime Type and set it
+			String mimeType = getMimeTypeFromFile(file);
```

Comment:
If mimeType is null, shouldn't the default be like it was before this change?

Reply:
I will replace this as well. It's the same thing as I explained above, I'm not sure if the old code is even able to return something more accurate than probeContentType(), but you're right, it is better to keep it just in case.